hack/helm_vars/wire-server/values.yaml.gotmpl

```diff
@@ -82,6 +82,7 @@ brig:
         - domain: federation-test-helper.{{ .Release.Namespace }}.svc.cluster.local
           search_policy: full_search
       set2FACodeGenerationDelaySecs: 5
+      setNonceTtlSecs: 5
```

Comment:
So if the customers don't change this, on-prem setup will be 5 seconds, not 5 minutes?  I'd prefer 5 minutes.